docs/topics/jvm-api-guidelines-readability.md

```diff
@@ -235,7 +235,7 @@ fun map(transform: (T) -> R): List<R>
 fun mapNotNull(transform: (T) -> R?): List<R>
 ```    
 
-It was possible to add something like `map(filterNulls: Boolean)` and write code like this:
+It is possible to add something like `map(filterNulls: Boolean)` and write code like this:
```

Comment:
Here we wanted to put the following meaning: when creating the library, it was possible to add a function `map(filterNulls: Boolean)`, but we didn't do in favor of `mapNotNull()`.